lib/recurly/risk/three-d-secure/three-d-secure.js

```diff
@@ -66,11 +71,17 @@ export class ThreeDSecure extends RiskConcern {
    * @param {HTMLElement} container
    */
   attach (container) {
-    this.whenReady(() => this.strategy.attach(container));
+    this.whenReady(() => {
+      this.strategy.attach(container);
+      this.report('attach');
+    });
   }
 
   remove () {
-    this.whenReady(() => this.strategy.remove());
+    this.whenReady(() => {
+      this.strategy.remove()
+      this.report('remove');
```

Comment:
I debated placing these reporters in `RiskConcern` but with the readiness need it made more sense to place it here. When we in the future move our other `Fraud` concerns over to the `RiskConcern` class, we can re-assess the SOC.